app/src/home.js

```diff
@@ -242,7 +244,79 @@ export default function Home() {
     });
   }, [ ])
 
+  const handleSave = (e) => {
+    e.preventDefault()
+    saveSession()
+  }
+
+  async function createSession(data) {
+    const apiName = 'MTADemoAPI';
+    const path = 'createSession';
+    const myInit = { 
+    //   headers: { 
+    //     Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}`,
+    //   },
+      response: true,
+      queryStringParameters: data
+    };
+
+    const result =  await API.post(apiName, path, myInit); 
+
+    return result;
+  }
 
+  const saveSession = () => {
+    const id = uuid();
+    Storage.configure({
+      bucket: 'mtastack-mtastackstorages3bucketc161f3b3-1tfncqzctldb1',
```

Comment:
Theese are hardcoded values. These should be taken from the property file and stored in index.js

For example, the UI would like : 

https://github.com/aws-samples/medical-transcription-analysis/blob/601e148b8ade294f5d3722a81b76f7d984b243ea/app/src/index.js#L14


You would need to pass the info between  deployment of CDK stack and build of the UI 
You need to add the s3 bucket resource here : https://github.com/aws-samples/medical-transcription-analysis/blob/master/app/bin/pre-build.js#L66-L98

Out other solution -DUS also passes S3 bucket and info in a similar way : https://github.com/awslabs/document-understanding-solution/blob/master/source/bin/pre-build.js#L69

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/eafae50489bbda5b2cd751d3679ef7a7ae8d11ce/app/src/home.js#L274-L277